Automattic/wp-calypso
# Jetpack section: Add Backup and Scan cards to the Business congratulation page

#### Changes proposed in this Pull Request

* Include a card for Backup and Scan in the congratulation page we show when users upgrade to Business plan.

#### Testing instructions

You can test this PR by going directly to the URL of the congratulation page (as the instructions below indicate), or you can simulate the whole purchasing process. If you want to do the latter, you'll need a WPCOM site that can be upgraded to a Business plan.

* **Prerequisite**: WPCOM site with Free, Personal, or Premium plan.
* Run this PR (Calypso).
* Visit `http://calypso.localhost:3000/plans/<your-site-slug>?customerType=business`.
* Upgrade your site to Business.
* (You should see the Congratulations page.)
* Verify that the Backup card is not present.
* Verify that the Scan card is not present.
* Add to the URL the `flags=jetpack/features-section` query parameter to activate the Jetpack section.
* Reload the page.
* Verify that the Backup card is present and its link points to the new Backup section.
* Verify that the Scan card is present and its link points to the new Scan section.

Fixes 1179060693083348-as-1179702911174266

#### Demo
##### `jetpack/features-section` flag enabled.
![image](https://user-images.githubusercontent.com/3418513/84430595-47097100-ac00-11ea-8972-6b40e146f2d7.png)

##### `jetpack/features-section` flag disabled.
![image](https://user-images.githubusercontent.com/3418513/84430611-4ffa4280-ac00-11ea-8cfc-7cd132197e0d.png)
